packages/utils/src/unwrapErrorHandler.ts

```diff
@@ -0,0 +1,25 @@
+import isPlainObject from 'lodash/isPlainObject';
+
+import { ErrorSchema, FormValidation, GenericObjectType } from './types';
+
+/** Unwraps the `errorHandler` structure into the associated `ErrorSchema`, stripping the `addError()` functions from it
+ *
+ * @param errorHandler - The `FormValidation` error handling structure
+ * @returns - The `ErrorSchema` resulting from the stripping of the `addError()` function
+ */
+export default function unwrapErrorHandler<T = any>(errorHandler: FormValidation<T>): ErrorSchema<T> {
```

Comment:
If I were to rename the `createErrorHandler()` to `wrapWithErrorHandler()` then I wonder if calling this `unwrapFromErrorHandler()` is a bit better? or perhaps this could want to be renamed to be the analog of `create`... like `destroyErrorHandler()`? Hmmm, naming things is sometimes so challenging.

Reply:
I don't think this function needs to have an "opposite" name since it's not a true inverse (create is `TFormData -> FormValidation<T>` and unwrap is `FormValidation<T> -> ErrorSchema<T>`). I'm actually ok with the current names, but not opposed to changing them if you think it's easier to understand.